### PR TITLE
Fixing issue 551

### PR DESCRIPTION
**Pull request by sbezverk**
Closes: https://github.com/kubernetes-incubator/external-storage/issues/551

### CODE REVIEW

File: local-volume/helm/provisioner/templates/provisioner.yaml [hunk 1]
```diff
@@ -71,23 +90,15 @@ spec:
...
-   {{- range $storageClass, $classConfig := $val }}
-        - name: {{ $storageClass }}
-          hostPath:
-            path: {{ $classConfig.hostDir }}
-   {{- end}}
-{{- end}}
+{{- if eq ( $engine | lower ) "gcepre19" }} {{ include "configVolumes" (.Values.configmap.gcePre19.storageClass) | indent 8 }} {{ end }}
+{{- if eq ( $engine | lower ) "gcepost19" }} {{ include "configVolumes" (.Values.configmap.gcePost19.storageClass) | indent 8 }} {{ end }}
+{{- if eq ( $engine | lower ) "gke" }} {{ include "configVolumes" (.Values.configmap.gke.storageClass) | indent 8 }} {{ end }}
+{{- if eq ( $engine | lower ) "baremetal" }} {{ include "configVolumes" (.Values.configmap.baremetal.storageClass) | indent 8 }} {{ end }}
```
Comment (msau42) [inline local-volume/helm/provisioner/templates/provisioner.yaml]: Where's the service account?

Comment (sbezverk) [reply local-volume/helm/provisioner/templates/provisioner.yaml]: https://github.com/kubernetes-incubator/external-storage/blob/master/local-volume/helm/provisioner/templates/provisioner.yaml#L58

Comment (msau42) [reply local-volume/helm/provisioner/templates/provisioner.yaml]: Thanks i'm blind

File: local-volume/helm/provisioner/templates/provisioner.yaml [hunk 2]
```diff
@@ -2,9 +2,28 @@
 {{- $classPath := . }}
 {{- range $tmp, $val := $classPath }}
    {{- range $storageClass, $classConfig := $val }}
-      {{ $storageClass }}:
-          hostDir: {{ $classConfig.hostDir }}
-          mountDir: {{ if $classConfig.mountDir }} {{$classConfig.mountDir }} {{ else }} {{ $classConfig.hostDir }} {{ end }}
+{{ $storageClass }}:
```
Comment (msau42) [inline local-volume/helm/provisioner/templates/provisioner.yaml]: Can we keep the indentation?

**Pull request merged**